### ISSUE TITLE
Recipe for matplotlib

### ISSUE THREAD

**Issue by filipefigcorreia** [kivy/kivy-ios#208]
It'd be very useful to have a recipe for matplotlib.

I would be interested in contributing one, but I'm not sure where to start, as I'm very new to iOS and to Kivy. I welcome any pointers that anyone may be able to provide on creating recipes in general and on creating one for matplotlib in particular.
##

**Comment by inclement** [comment]
I don't know about ios specifically, but for android I anticipate this being relatively complex. Try starting by making a recipe for something simpler, e.g. one of the modules supported by python-for-android but not for kivy-ios yet.

**Comment by filipefigcorreia** [comment]
Thanks for the suggestion. I will start by creating a recipe for `libpng` then, since it's one of the dependencies of `matplotlib`. The [`libpng` recipe for Android](https://github.com/kivy/python-for-android/blob/master/pythonforandroid/recipes/png/__init__.py) looks (suspiciously) simple.

**Comment by filipefigcorreia** [comment]
I have started with a very simple recipe for matplotlib but it is currently failing. Any help appreciated in figuring out why.

There are a few of these warnings, which I'm not sure should be a concern:

    Could not find platform dependent libraries <exec_prefix>
    Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]

And then it seems to fail for not finding a fortran compiler, which is odd, since I do have a fortran compiler (`gfortran`) which I've installed with `brew install gcc`.

A more complete console output can be found here:
https://gist.github.com/filipefigcorreia/b7517ecfa564744143dbc8152e1088aa
